opensearch-operator/examples/opensearch-cluster.yaml

```diff
@@ -39,7 +40,7 @@ spec:
         - "data"
     - component: nodes
       replicas: 3
-      diskSize: "100Gi"
+      diskSize: "30Gi"
```

Comment:
i'd strongly suggest to work with atomic commits, otherwise the history is impossible to understand (and it makes it a lot harder to use `git revert`, `git bisect`, etc.)